commit 6795f0fd525e33751b7cae77ff18532616dbb89c
Author: QbProg <tholag@gmail.com>
Date:   Sun Jun 19 18:20:09 2011 +0200

    Marked @todos related to some of the MSVC warnings

From 23ca5ad4f8a3c6972e7f60b8c4a0fb04a017cb4a Mon Sep 17 00:00:00 2001
From: Hugues Delorme <delorme.hugues@fougsys.fr>
Date: Mon, 22 Aug 2011 18:04:31 +0200
Subject: [PATCH] [warning-fix][comparison-between-different-enums]
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In module TKOpenGL, fixed the two remaining GCC warnings of the form :
  "comparison between ‘enum texStatus’ and ‘enum texDataStatus’"
Obviously texdata[data].status is wanted instead of textab[ID].status.
  enum texDataStatus conveys the dimensional of the texture (1D or 2D),
  whereas enum texStatus only conveys the "allocated" information.
---
 src/OpenGl/OpenGl_TextureBox.cxx |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

Index: oce/inc/Prs2d_Dimension.hxx
===================================================================
--- oce.orig/inc/Prs2d_Dimension.hxx
+++ oce/inc/Prs2d_Dimension.hxx
@@ -122,7 +122,8 @@ public:
         Standard_Boolean ArrowIsReversed() const;
   //! Return true if symbol is drawn, false - in otherwise <br>
         Standard_Boolean IsDrawSymbol() const;
-  //! Return ASCII code of the symbol <br>
+  //! Return ASCII code of the symbol <br> 
+  // @todo Should really be Standard_Character
         Standard_Integer SymbolCode() const;
   
   Standard_EXPORT    const Handle_TColgp_HArray1OfPnt2d ArrayOfFirstArrowPnt() const;
Index: oce/src/IFSelect/IFSelect_WorkSession.cxx
===================================================================
--- oce.orig/src/IFSelect/IFSelect_WorkSession.cxx
+++ oce/src/IFSelect/IFSelect_WorkSession.cxx
@@ -2822,7 +2822,7 @@ void IFSelect_WorkSession::QueryCheckLis
 {
   if (!IsLoaded()) return;
   Standard_Integer i,nb = themodel->NbEntities();
-  thecheckana = TCollection_AsciiString (' ',nb+1);
+  thecheckana = TCollection_AsciiString (' ',nb+1); // @todo Probably the constructor here has to be called with inversed parameters (nb+1,' ')
   for (chl.Start(); chl.More(); chl.Next()) {
     Standard_Integer num = chl.Number();
     const Handle(Interface_Check) ach = chl.Value();
